docs/source/reference/series.rst

```diff
@@ -157,15 +157,15 @@ String    :ref:`str <api.series.str>`
 
 .. _api.series.dt:
 
-Datetimelike Properties
-~~~~~~~~~~~~~~~~~~~~~~~
+Date Time Handling
```

Comment:
This also was from Pandas (https://pandas.pydata.org/pandas-docs/stable/reference/series.html#datetimelike-properties)